docs/calico-with-docker/without-docker-networking/vagrant-ubuntu/Vagrantfile

```diff
@@ -53,10 +53,6 @@ Vagrant.configure(2) do |config|
       host.vm.provision :shell, inline: "wget -qO /usr/local/bin/calicoctl #{calicoctl_url}", :privileged => true
       host.vm.provision :shell, inline: "chmod +x /usr/local/bin/calicoctl"
 
-      # Replace docker with the docker 1.10.1 build
-      host.vm.provision :shell, inline: "stop docker", :privileged => true
-      host.vm.provision :shell, inline: "wget -qO /usr/bin/docker https://get.docker.com/builds/Linux/x86_64/docker-1.10.1", :privileged => true
-
```

Comment:
This was failing before (see #892 from the description above) because we were not starting docker after shutting it down.  This was the original reason for making this change.